pds/server.go

```diff
@@ -563,15 +510,15 @@ func (s *Server) lookupUserByHandle(ctx context.Context, handle string) (*User,
 
 func (s *Server) userCheckMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
 	return func(c echo.Context) error {
-		ctx := c.Request().Context()
+		req := c.Request()
+		ctx := req.Context()
 
-		user, ok := c.Get("user").(*gojwt.Token)
-		if !ok {
+		token := getToken(ctx)
```

Comment:
This needs double-checking. The original code was confusing the `Token` types from two different `jwt` modules, and possibly confusing users with tokens.